Binance Detect Moonings.py

```diff
@@ -319,30 +331,36 @@ def sell_coins():
         # check that the price is above the take profit and readjust SL and TP accordingly if trialing stop loss used
         if float(last_price[coin]['price']) > TP and USE_TRAILING_STOP_LOSS:
             print("TP reached, adjusting TP and SL accordingly to lock-in profit")
-            
+
             # increasing TP by TRAILING_TAKE_PROFIT (essentially next time to readjust SL)
             coins_bought[coin]['take_profit'] += TRAILING_TAKE_PROFIT
             coins_bought[coin]['stop_loss'] = coins_bought[coin]['take_profit'] - TRAILING_STOP_LOSS
 
             continue
 
         # check that the price is below the stop loss or above take profit (if trailing stop loss not used) and sell if this is the case 
-        if float(last_price[coin]['price']) < SL or (float(last_price[coin]['price']) > TP and not USE_TRAILING_STOP_LOSS):
-            print(f"{txcolors.SELL}TP or SL reached, selling {coins_bought[coin]['volume']} {coin} - {BuyPrice} - {LastPrice} : {PriceChange:.2f}%{txcolors.DEFAULT}")
+        if float(last_price[coin]['price']) < SL or (
+                float(last_price[coin]['price']) > TP and not USE_TRAILING_STOP_LOSS):
+            print(
+                f"{txcolors.SELL}TP or SL reached, selling {coins_bought[coin]['volume']} {coin} - {BuyPrice} - {LastPrice} : {PriceChange:.2f}%{txcolors.DEFAULT}")
 
-            if TESTNET :
+            if TESTNET:
                 # create test order before pushing an actual order
-                test_order = client.create_test_order(symbol=coin, side='SELL', type='MARKET', quantity=coins_bought[coin]['volume'])
+                test_order = client.create_test_order(symbol=coin, side='SELL', type='MARKET',
+                                                      quantity=coins_bought[coin]['volume'])
 
             # try to create a real order if the test orders did not raise an exception
             try:
-
+                sell_amount = coins_bought[coin]['volume']
+                tick_size = float(next(
+                    filter(lambda f: f['filterType'] == 'LOT_SIZE', client.get_symbol_info(coin)['filters'])
```

Comment:
maybe you can cache lot_size into coins_bought.json

Reply:
Makes perfect sense. I'll implement it.